src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let currentSum = 0;
+
+  const adder = function (num) {
+    if (num === undefined) {
+      const result = currentSum;
+
+      currentSum = 0;
+
+      return result;
+    } else {
+      currentSum += num;
```

Comment:
Adding numbers to currentSum is best done in a loop for all passed arguments.

Example:
```suggestion
      for (let num of args) {
        currentSum += num;
      }
```